clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -3160,6 +3160,23 @@ static void handleWorkGroupSize(Sema &S, Decl *D, const ParsedAttr &AL) {
   S.addIntelTripleArgAttr<WorkGroupAttr>(D, AL, XDimExpr, YDimExpr, ZDimExpr);
 }
 
+// The lambda returns a DupArgResult value; Same means the args have the same
```

Comment:
Should this comment be updated? `AreArgValuesIdentical` is not lambda anymore.

Reply:
Good catch, I've updated.